docs/trace_analytics.md

```diff
@@ -33,7 +33,7 @@ The [OpenTelemetry source](../data-prepper-plugins/otel-trace-source/README.md)
 ### Processor
 
 We have two processor for the Trace Analytics feature,
-* *otel_trace_raw_prepper* -  This processor is responsible for converting the trace data in [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-proto/tree/master/opentelemetry/proto/trace/v1) to OpenSearch-friendly (JSON) docs. These OpenSearch-friendly docs have certain additional fields like duration which are not part of the original OpenTelemetry specification. These additional fields are to make the instant OpenSearch Dashboards dashboards user-friendly.
+* *otel_trace_raw* -  This is a processor that fills in the missing trace group related fields in the collection of [Span](../data-prepper-api/src/main/java/com/amazon/dataprepper/model/trace/Span.java) records output by [otel_trace_raw](../otel-trace-raw-processor) processor.
```

Comment:
These descriptions are entirely different. Has our plugin changed that much in terms of functionality with the model change?

Reply:
Part of the reason is that the previous doc was also outdated:

Previous:
1. converting OTLP trace data to json docs
2. fill-in missing trace group related fields

Now: only 2 is performed since we are already passing in event model

Reply:
This seems to be the description for `otel_trace_group` instead of `otel_trace_raw`.
By the way, does `otel_trace_group` belong to this section as well?

Reply:
Thanks for the catch! I will add both processors here as well as their descriptions